tests/unit/utils.js

```diff
@@ -0,0 +1,60 @@
+ const assert = require('chai').assert;
+
+// create a global window so we can unittest the window.<x> functions
+if (!global.window)
+   global.window = new Object({});
+
+require('../../saltgui/static/scripts/utils');
+
+
+describe('Unittests for utils.js', function() {
```

Comment:
Some tests for potential mistakes would be great here (one _minutes_ etc.)

Reply:
Yeah well, it would be fantastic but this function is now tested enough I think. PR's are welcome 😃